hyperon_das_atomdb/adapters/ram_only.py

```diff
@@ -675,11 +681,7 @@ def add_node(self, node_params: NodeParamsT) -> NodeT | None:
         return node
 
     def add_link(self, link_params: LinkParamsT, toplevel: bool = True) -> LinkT | None:
```

Comment:
Return annotation is not optional anymore:
```suggestion
    def add_link(self, link_params: LinkParamsT, toplevel: bool = True) -> LinkT:
```

Reply:
Fixed